src/main/java/uk/gov/hmcts/reform/bulkscan/payment/processor/controllers/PaymentController.java

```diff
@@ -0,0 +1,58 @@
+package uk.gov.hmcts.reform.bulkscan.payment.processor.controllers;
+
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.responses.ApiResponse;
+import io.swagger.v3.oas.annotations.tags.Tag;
+import jakarta.validation.Valid;
+import org.springframework.context.annotation.Profile;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import uk.gov.hmcts.reform.bulkscan.payment.processor.models.CreatePayment;
+import uk.gov.hmcts.reform.bulkscan.payment.processor.models.UpdatePayment;
+import uk.gov.hmcts.reform.bulkscan.payment.processor.service.PaymentService;
+
+@RestController
+@RequestMapping("/payment")
+@Profile("!functional")
```

Comment:
If we want to have functional tests for this, we may need to remove this. Can probably ignore if not